Magic/2. The Art of Magic.md

```diff
@@ -0,0 +1,30 @@
+# 2. Art of magic
+
+## The learning
+An uninitiated who don't open his spirit can't feel the praeter, even when he's near a place of power.
+Therefor, becoming a wizard is a long journey of pilgrimage to places of power to learn to open his spirit to feel and get used to the praeter. It's nothing like book reading.
+
+Once the initiated reach a certain level, he can enter the praeter properly. Then he must remain in these low bonds sources of power to progressively reach the level where he can use the praeter to cast spells.
+
+The use of strong bonds sources of power requires a way more important training and is reserved for talented wizards.
+
+The strength and the length of a spell are limited by both the caster and, if used, the source of power.
+Performing magic is exhausting for the caster, as draining energy from the praeter requires a great concentration. Although it can't injure the caster physically, it still can faint him in extreme cases.
+
+Casting does not require any gesture, lyrics or ritual.
+
+## The sources of power
+All the sources of powers allow the caster to perform any kind of magic. There are two things to mention though: first each kind of spells need specific trainings to master, and secondly each kind of magic requires a specific environment to be performed.
```

Comment:
I detailed in the third part [link](https://github.com/Adkorh/univers/blob/patch-1/Magic/3.%20The%20Frame%20of%20Magic.md)